docs/platform-specification.md

```diff
@@ -625,6 +644,51 @@ The tool configuration properties are available globally without the prefix. For
 property can be used as **{cmd.path}** inside the recipe, and the same happens for all the other avrdude configuration
 variables.
 
+#### Pluggable Discovery
```

Comment:
```suggestion
#### Pluggable discovery
```
Even though it could be argued that [title case](https://en.wikipedia.org/wiki/Title_case) rules should apply here, most of the document's section headings use sentence case and I think that it's best to follow the established style convention.